src/test/java/ch/uzh/ifi/hase/soprafs23/service/UserServiceTest.java

```diff
@@ -198,7 +196,7 @@ public void testUpdateUserWithExistingUsername() {
         userPutDTO.setUsername(newUsername);
 
         when(userRepository.findById(userId)).thenReturn(Optional.of(existingUser));
-        when(userRepository.findByUsername(newUsername)).thenReturn(new User());
+        when(userRepository.findByUsername(newUsername)).thenReturn(new User()); //this was before: new User()
```

Comment:
seeing the changes side by side: you only added the comment here ^^'

Reply:
cleaned it up